hknweb/academics/serializers/course_surveys_serializers.py

```diff
@@ -0,0 +1,21 @@
+from rest_framework import serializers
+
+from ..models.course_surveys import Question, Rating, Survey
+
+
+class QuestionSerializer(serializers.ModelSerializer):
```

Comment:
Have you looked into `HyperlinkedModelSerializer`s? They use hyperlinks for foreign and primary keys instead of the key values, which would be convenient because we wouldn't need to explicitly specify the URLs in the frontend to these entities. What do you think?

Reply:
Done.